.github/workflows/release.yaml

```diff
@@ -188,9 +188,9 @@ jobs:
     strategy:
       matrix:
         minor:
-          - '20'
```

Comment:
I believe that we still want to release and test for 1.20 so I'd leave this here.

Reply:
fixed.